packages/react-button/jest.config.js

```diff
@@ -1,10 +1,12 @@
-const { createConfig, resolveMergeStylesSerializer } = require('@fluentui/scripts/jest/jest-resources');
 const path = require('path');
 
-const config = createConfig({
+module.exports = {
+  displayName: 'react-button',
+  preset: '../../jest.preset.js',
+  transform: {
+    '^.+\\.[tj]sx?$': ['babel-jest', { cwd: __dirname, configFile: './babel-jest.config.json' }],
+  },
+  moduleFileExtensions: ['ts', 'tsx', 'js', 'jsx'],
```

Comment:
```suggestion
  transform: {
    '^.+\\.tsx?$': ['babel-jest', { cwd: __dirname, configFile: './babel-jest.config.json' }],
  },
  moduleFileExtensions: ['ts', 'tsx'],
```

nit: I don't think that we plan to we have `.js`/`.jsx` files in repo as test targets